Sources/Kitura/Kitura.swift

```diff
@@ -212,16 +219,16 @@ public class Kitura {
     /// Make all registered servers start listening on their port.
     ///```swift
     /// let router = Router()
-    /// Kitura.addHTTPServer(onPort: 8080, with: router)
+    /// Kitura.addHTTPServer(onPort: 8080, with: router, address: "localhost")
     /// Kitura.startWithStatus() // Returns the number of failed server starts.
     ///```
     public class func startWithStatus() -> Int {
         serverLock.lock()
         var numberOfFailures = 0
-        for (server, port) in httpServersAndPorts {
-            Log.verbose("Starting an HTTP Server on port \(port)...")
+        for (server, port, address) in httpServersAndPorts {
+            Log.verbose("Starting an HTTP Server on port \(port) with address \(String(describing: address))...")
```

Comment:
I _think_ this will result in `Optional(value)`, can I suggest using a default value (eg. '*') to avoid this.
Example in the version I mocked up here: a5539cf103ea1e3047b23d64b633160b091b4c4f